packages/contract-method-name/cfx-name.js

```diff
@@ -19,16 +31,17 @@ export const getCFXAbi = async (address, networkType) => {
 export const getCFXContractMethodSignature = async (
   address,
   transactionData,
-  networkType,
+  network,
 ) => {
   try {
-    const response = await getCFXAbi(address, networkType)
-    const abi = JSON.parse(response.abi)
-    return new Conflux()
-      .Contract({abi, address})
-      .abi.decodeData(transactionData)
+    let abi = []
+    if (eip777AbiSignatures.includes(transactionData.substr(0, 10))) {
+      return cfxContractInterface.abi.decodeData(transactionData)
```

Comment:
@Pana Need help check if this function call request `networkId` to be set

Reply:
If contractInstance's conflux attribute have networkId info, it will decode address type to base32, If not it will return address in hex format

Reply:
done